hooks/queries/votes/usePastVotes.ts

```diff
@@ -2,15 +2,22 @@ import { useQuery } from "@tanstack/react-query";
 import { pastVotesKey } from "constants/queryKeys";
 import { getPastVotes } from "graph";
 import { useHandleError, useVoteTimingContext } from "hooks";
+import { usePaginationContext } from "hooks/contexts/usePaginationContext";
 
 export function usePastVotes() {
   const { roundId } = useVoteTimingContext();
   const onError = useHandleError();
+  const {
+    pageStates: { pastVotesPage },
+  } = usePaginationContext();
 
-  const queryResult = useQuery([pastVotesKey, roundId], () => getPastVotes(), {
+  const { resultsPerPage, number } = pastVotesPage;
+
+  const queryResult = useQuery([pastVotesKey, roundId, pastVotesPage], () => getPastVotes(resultsPerPage, number), {
     refetchInterval: (data) => (data ? false : 100),
     initialData: {},
     onError,
+    keepPreviousData: true,
```

Comment:
this prevents react query from showing a loading state when paginating, and instead we show the old data until the new is ready